ssv/spectest/tests/runner/duties/newduty/duplicate_duty_finished.go

```diff
@@ -0,0 +1,65 @@
+package newduty
+
+import (
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// DuplicateDutyFinished is a test that runs the following scenario:
+// - Runner is assigned a duty
+// - Runner finishes the duty
+// - Runner is assigned the same duty again
+// TODO - Does it make sense that the runner starts? Does it make sense that duties with precon phase don't return an error?
+func DuplicateDutyFinished() *MultiStartNewRunnerDutySpecTest {
```

Comment:
This Test was added since the original Finished test seems to be trying to cover more than one scenario.

What is important here that the runner finishes and then a duty of a similar height or less appears. It doesn't have to be duplicate so maybe we can change the test.

Reply:
opened https://github.com/bloxapp/ssv-spec/issues/282